azurerm/internal/services/databasemigration/database_migration_project_resource_test.go

```diff
@@ -110,8 +110,6 @@ func (t DatabaseMigrationProjectResource) Exists(ctx context.Context, clients *c
 }
 
 func (DatabaseMigrationProjectResource) basic(data acceptance.TestData) string {
```

Comment:
we could make this `r`:

```suggestion
func (r DatabaseMigrationProjectResource) basic(data acceptance.TestData) string {
```